src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -106,5 +108,26 @@ public static void DeleteHitsFile(string path)
 
             RetryHelper.Retry(() => File.Delete(path), retryStrategy, 10);
         }
+        
+        public static string[] GetExcludedFiles(string[] exclusionRules, string parentDir) {
+            if (exclusionRules == null || exclusionRules.Length == 0 ) return null;
```

Comment:
IMHO usually i prefer `Array.Empty<string>()` https://docs.microsoft.com/en-us/dotnet/api/system.array.empty?view=netcore-2.0 less check condition for users.